sdk/computeschedule/arm-computeschedule/package.json

```diff
@@ -68,7 +68,7 @@
     "@types/node": "^18.0.0",
     "eslint": "^8.55.0",
     "prettier": "^3.2.5",
-    "typescript": "~5.5.3",
+    "typescript": "~5.6.2",
```

Comment:
this is to revert some previous pr has changed the typescript unintentionally by code generation.